src/recap

```diff
@@ -640,90 +569,85 @@ check_disk_space() {
 
 # check to see where the configuration file is located.
 # if the file is not at /etc/recap, make some noise to alert users
-if [ -f /etc/sysconfig/recap ] && [ -f /etc/recap ]
+if [ -r /etc/sysconfig/recap ] && [ -r /etc/recap ]
 then
 	echo "Configuration files exist at old (/etc/sysconfig/recap) and new locations (/etc/recap). The file from the old location will be read. Please consolidate your configuration details into /etc/recap."
-	. /etc/sysconfig/recap
-elif [ -f /etc/sysconfig/recap ] && [ ! -f /etc/recap ]
+	source /etc/sysconfig/recap
+elif [ -r /etc/sysconfig/recap ] && [ ! -r /etc/recap ]
 then
 	echo "Configuration file exists at old location (/etc/sysconfig/recap). The file will be read. Please move your configuration file to /etc/recap."
-	. /etc/sysconfig/recap
-elif [ ! -f /etc/recap ]
+	source /etc/sysconfig/recap
+elif [ ! -r /etc/recap ]
 then
 	echo "No configuration file found. Proceeding with defaults."
 else
-	. /etc/recap
+	source /etc/recap
 fi
 
 # Quit if load is higher than MAXLOAD
-LOAD=$(uptime | awk -F'load average:' '{ print $2 }' | cut -d" " -f2 | cut -d"," -f1)
-QUIT=$(echo $LOAD'>'$MAXLOAD | bc -l)
-if [ $QUIT -eq 1 ]
-then
-    echo "QUIT (LOAD:"$LOAD", MAXLOAD:"$MAXLOAD")"
-    exit 1
+LOAD=$( grep -Po '^[\d.]+' /proc/loadavg )
```

Comment:
Inside square brackets(character sets) the `.` represents a `"dot"` the purpose of the square bracket is to match a single char with a specific pattern, examples
-  `[\d]` a digit
- `[a-z]` a range
- `[YyNn]` a y/n option
- `[etc]` - `e` or `t` or `c` :P

I agree at first sight seems confusing, but is valid, I'll make the change.

See this description of the perl regex flavor syntax for the character sets, while is not explicit, it is implied that the use of `[]` is to evaluate matching patters to a single char, if `[.]` meant any char, the purpose of its use would be null as that can be written `.`. http://www.boost.org/doc/libs/1_50_0/libs/regex/doc/html/boost_regex/syntax/perl_syntax.html#boost_regex.syntax.perl_syntax.character_sets

There shouldn't be any issue making the change:

Matching a dot or more:
```bash
  $ echo -e "1.23\n1_23\n1/23\n1.2.3" | grep -P "[.]+"
  1.23
  1.2.3

  $ echo -e "1.23\n1_23\n1/23\n1.2.3" | grep -P "[\.]+"
  1.23
  1.2.3
```